package.json

```diff
@@ -105,10 +114,5 @@
   "lint-staged": {
     "*.js": "vjsstandard --fix",
     "README.md": "doctoc --notitle"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
```

Comment:
does this not work with newer husky versions?

Reply:
I'll retest that

Reply:
(ok I put it back)